exercises/code_structure/plotting_solution.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Styles for markers and lines
+colors = ['green', 'blue', 'purple', 'red']
+linestyles = ['--', '-', '-.', ':']
+markers = ['o', 'x', 'v', '*']
```

Comment:
I really like the idea of setting this up like this and haven't come across it before!